ground/gcs/src/plugins/gcscontrolwidget/gcscontrolgadget.cpp

```diff
@@ -125,7 +125,7 @@ GCSControl* GCSControlGadget::getGcsControl() {
 void GCSControlGadget::manualControlCommandUpdated(UAVObject * obj) {
 
     // Not sending then show updates from transmitter
-    if (!enableSending)
+    if (enableSending)
```

Comment:
You mind trying to rebase and force push your branch? Looks like we are getting some changes from a previous commit in this diff.